pkg/render/common/statik/statikfiles.go

```diff
@@ -0,0 +1,37 @@
+package statik
+
+import (
+	"fmt"
+	"io/ioutil"
+
+	"github.com/rakyll/statik/fs"
+	_ "github.com/tigera/operator/statik"
+)
+
+// getStatikFile gets the static file embeded into the binary using statik library (https://github.com/rakyll/statik).
+// All static files are located in statik/files. statik/statik.go is the source file that is generated
+// from the static files. It can be recreated using the following command.
+// "statik -src=/home/suresh/go/src/github.com/tigera/operator/statik/files"
```

Comment:
I didn't understand, I thought you were adding something that would include a file at run time.
If this requires a separate step it should be containerized and ran as part of the build process.